nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -986,3 +982,54 @@ content initially copied from the community wiki. The wiki enhances the official
 existing resources and providing a categorization system for easy navigation, and is guided by a new "Manual
 of Style" — a contribution guide and enhanced templates. It offers a wealth of new information, including
 articles on applications, desktop environments, and a growing number of translations in multiple languages.
+
+## Nixpkgs Library {#sec-release-24.11-lib}
```

Comment:
If you have the time: I think placing this above the wiki section could be better, as it's listing internal changes similar to the regular NixOS/pkgs notes

Reply:
Not sure I get the reason, the `lib` changes aren't internal

Reply:
That was a poor choice of words. What I meant is they are changes directly relating to the release, with additions, breakages, etc. being listed that will immediately affect users. I think it would be nice to keep those together with the NixOS equivalents, rather than have them split by what is more like a general update on a complementary resource

Reply:
Ah I see, sure!